### PR TITLE
fix: fix the license issues

### PR DESCRIPTION
**Pull request by mindlesscloud**
### Summary
- remove pagerduty from the `requirements.txt`
- update the LICENSE file
- update the NOTICE file and supply the following information:

>  - [github.com/hashicorp/hcl] (https://github.com/hashicorp/hcl), licensed under the Mozilla Public License 2.0 License
> - [github.com/davecgh/go-spew] (https://github.com/davecgh/go-spew), licensed under the ISC License
> - [github.com/russross/blackfriday] (https://github.com/russross/blackfriday), licensed under the Simplified BSD License
> - [github.com/gogo/protobuf] (https://github.com/gogo/protobuf), licensed under the BSD License 2.0
> - [github.com/emirpasic/gods] (https://github.com/emirpasic/gods), licensed under the Simplified BSD License
> - [github.com/go-git/gcfg] (https://github.com/go-git/gcfg), licensed under the BSD License 2.0
> - [github.com/kevinburke/ssh_config] (https://github.com/kevinburke/ssh_config), licensed under the MIT License
> - [github.com/pmezard/go-difflib] (https://github.com/pmezard/go-difflib), licensed under the BSD License 2.0

### CODE REVIEW

**Review by klesh — APPROVED** [review]
LGTM

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,2 @@
 setuptools
-wheel
-tap-pagerduty==0.2.0
```
Comment (keon94) [inline requirements.txt:3]: why is this deleted?

Comment (mindlesscloud) [reply requirements.txt:3]: > why is this deleted?

For the sake of compliance with the Apache release policy
